[{"ContainerToolsException.Message":"No se pudo detener la aplicación en el contenedor. Puede que la compilación genere un error porque los archivos están en uso.\r\nNo se pudo ejecutar el comando de Docker. Código de salida: 0.\r\nsh: 0: getcwd() failed: No such file or directory","ContainerToolsException.ErrorCode":"CTP1006","ContainerToolsException.StackTrace":"   en Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<KillAppInsideContainerAsync>d__4.MoveNext()\r\n--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---\r\n   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   en Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildTask.<ExecuteAsync>d__24.MoveNext()","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.InnerException":"Microsoft.VisualStudio.Containers.Tools.Common.ContainerToolsException: No se pudo ejecutar el comando de Docker. Código de salida: 0.\r\nsh: 0: getcwd() failed: No such file or directory\r\n   en Microsoft.VisualStudio.Containers.Tools.Common.Client.DockerClient.<ExecAsync>d__13.MoveNext()\r\n--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---\r\n   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   en Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<KillAppInsideContainerAsync>d__4.MoveNext()\r\nCódigo de error: CTC1029\r\nGravedad: Error\r\nLínea de comandos: docker exec -i 7e25b1a95509193f8e68a346b1f7271d6e593f7db8bee19d89ff2987af1160fb /bin/sh -c \"if PID=$(pidof dotnet); then kill $PID; fi\"\r\nCódigo de salida del comando: 0\r\nSalida del comando: \r\nError de comando: sh: 0: getcwd() failed: No such file or directory\r\n"},{"ContainerToolsException.Message":"No se pudo detener la aplicación en el contenedor. Puede que la compilación genere un error porque los archivos están en uso.\r\nNo se pudo ejecutar el comando de Docker. Código de salida: 0.\r\nsh: 0: getcwd() failed: No such file or directory","ContainerToolsException.ErrorCode":"CTP1006","ContainerToolsException.StackTrace":"   en Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<KillAppInsideContainerAsync>d__4.MoveNext()\r\n--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---\r\n   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   en Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildTask.<ExecuteAsync>d__24.MoveNext()","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.InnerException":"Microsoft.VisualStudio.Containers.Tools.Common.ContainerToolsException: No se pudo ejecutar el comando de Docker. Código de salida: 0.\r\nsh: 0: getcwd() failed: No such file or directory\r\n   en Microsoft.VisualStudio.Containers.Tools.Common.Client.DockerClient.<ExecAsync>d__13.MoveNext()\r\n--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---\r\n   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   en Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<KillAppInsideContainerAsync>d__4.MoveNext()\r\nCódigo de error: CTC1029\r\nGravedad: Error\r\nLínea de comandos: docker exec -i 7e25b1a95509193f8e68a346b1f7271d6e593f7db8bee19d89ff2987af1160fb /bin/sh -c \"if PID=$(pidof dotnet); then kill $PID; fi\"\r\nCódigo de salida del comando: 0\r\nSalida del comando: \r\nError de comando: sh: 0: getcwd() failed: No such file or directory\r\n"}]